.github/workflows/continuous-deployment.yml

```diff
@@ -1,77 +1,77 @@
-# This is a basic workflow to help you get started with Actions
+# # This is a basic workflow to help you get started with Actions
```

Comment:
Can we change this to something more descriptive of what the workflow is doing?

Reply:
I have the GitHub workflow commented out for now. We have an issue in the backlog to re-enable and upgrade this deployment script.